app/pages/project/index.cjsx

```diff
@@ -87,8 +88,12 @@ ProjectPage = React.createClass
       @context.geordi?.remember projectToken: nextProps.project?.slug
 
     # Only call to get workflow if we know if there is a user or not and the project is finished loading
-    if nextContext.initialLoadComplete and not nextProps.loading and nextProps.preferences and @state.activeWorkflows.length is 0
-      @getAllWorkflows(nextProps.project)
+    if nextContext.initialLoadComplete and not nextProps.loading and nextProps.preferences and @state.activeWorkflows.length is 0 and not @state.loadingSelectedWorkflow
```

Comment:
Minor, I'd move this to a well named function so one doesn't have to interpret all the boolean state to understand this conditional